src/service/section/system-section-service.ts

```diff
@@ -247,12 +227,14 @@ export class SystemSectionService extends AbstractSectionService {
         return this.systemRepository.getDevices(deviceClass);
     }
 
-    public listAlertFilters() {
-        return this.alertFilterRepository.listAlertFilters();
-    }
     public saveAlertFilters(alertFilters: Array<AlertFilter>) {
         return _.map(alertFilters, alertFilter => this.alertFilterRepository.save(alertFilter));
     }
+
+    public deleteAlertFilters(alertFilters: Array<AlertFilter>) {
+        return _.map(alertFilters, alertFilter => this.alertFilterRepository.delete(alertFilter));
```

Comment:
To return the tasks submitted to the middleware, it's probably more clear once typed :smile: